samples/ipc/ipc_service/README.rst

```diff
@@ -54,40 +54,50 @@ Building and running
 
 .. include:: /includes/build_and_run.txt
 
+nRF5340 DK
+----------
+
+RPMsg and ICMSG backends can be tested using the following build commands:
+
+.. code-block:: console
+
+   west build -p -b nrf5340dk_nrf5340_cpuapp -T sample.ipc.ipc_service.nrf5340dk_rpmsg_default
```

Comment:
Such command gives "FATAL ERROR: test item path does not exist" because the source directory is not specified. It should be either `-T ./sample.ipc.ipc_service.nrf5340dk_rpmsg_default` or `-T sample.ipc.ipc_service.nrf5340dk_rpmsg_default .`.